src/parser/core/Events.ts

```diff
@@ -380,7 +390,7 @@ export interface BaseCastEvent<T extends string> extends Event<T> {
   sourceInstance?: number;
   sourceIsFriendly: boolean;
   spellPower?: number;
-  target?: { name: 'Environment'; id: -1; guid: 0; type: 'NPC'; icon: 'NPC' };
+  target?: CastTarget;
```

Comment:
hmm, strange that these were using a const interface like that. good catch. some of these legitimately don't ever have targets (`begincast` doesn't afaik) but `cast` certainly does